### ISSUE TITLE
Ergonomics: add a Jupyter notebook template for a processing step

### ISSUE THREAD

**Issue by larsyencken** [owid/etl#102]
Many people write Jupyter notebooks as ETL steps, but when you're getting started it's not obvious the style or structure you should use for them, and it can be a hassle to go look at other steps to find a good style.

If possible, we should add a template notebook that has a sensible layout and let people start from that, instead of a blank notebook.

**Comment by Marigold** [comment]
Agree, having templates would be really useful. I've started adding them into [`examples/` directory](https://github.com/owid/etl/tree/master/examples), but I'm starting to think that putting them into `etl/steps/data/examples` namespace would have been a better choice (especially since I can't reference other datasets like population in the `examples` directory). Any objections against putting them there and running them as part of the main pipeline?

**Comment by larsyencken** [comment]
I think the examples folder is excellent, and it sounds like the templates might be less necessary if more people are working in Python, since then they can browse to other examples much more easily. We can close this issue I think!
